core/shared/src/main/scala/zio/ZSchedule.scala

```diff
@@ -339,19 +345,30 @@ trait ZSchedule[-R, -A, +B] extends Serializable { self =>
     new ZSchedule[R1, A1, B] {
       type State = self.State
       val initial = f(self.initial)
+      val extract = self.extract
       val update  = self.update
     }
 
   /**
-   * Applies random jitter to the schedule bounded by the factors 0.0 and 1.0.
+   * Applies random jitter to all sleeps executed by the schedule.
    */
-  final def jittered: ZSchedule[R with Random, A, B] = jittered(0.0, 1.0)
+  final def jittered(min: Double = 0.0, max: Double = 1.0)(implicit ev: ZEnv <:< R): ZSchedule[ZEnv, A, B] =
+    jittered_(min, max)(f => ZEnv.mapClock(f) andThen ev.apply)
 
   /**
-   * Applies random jitter to the schedule bounded by the specified factors, with a given random generator.
+   * Applies random jitter to all sleeps executed by the schedule.
+   * This version supports arbitrary environments.
    */
-  final def jittered(min: Double, max: Double): ZSchedule[R with Random, A, B] =
-    modifyDelay((_, d) => nextDouble.map(random => d * min * (1 - random) + d * max * random))
+  final def jittered_[R1](min: Double, max: Double)(
```

Comment:
I don't feel this use of `_` is consistent with other usages, but I may be wrong. Maybe `jitteredEnv` or `jitteredWithEnv` or something?